### PR TITLE
[BZ2035368]: Explain non-STS clusters cannot switch to STS

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.7+

Issue:
[BZ2035368](https://bugzilla.redhat.com/show_bug.cgi?id=2035368)

Link to docs preview:
- [Using manual mode with STS](https://deploy-preview-44944--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-mode-sts)
- [Using manual mode with GCP Workload Identity](https://deploy-preview-44944--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.html#cco-mode-gcp-workload-identity)
- [Alternatives to storing administrator-level secrets in the kube-system project (AWS)](https://deploy-preview-44944--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/manually-creating-iam.html#alternatives-to-storing-admin-secrets-in-kube-system_manually-creating-iam-aws)
- [Alternatives to storing administrator-level secrets in the kube-system project (GCP)](https://deploy-preview-44944--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_gcp/manually-creating-iam-gcp.html#alternatives-to-storing-admin-secrets-in-kube-system_manually-creating-iam-gcp)

Additional information:
Adds note that existing clusters cannot be reconfigured to use either of the short-term, limited-privilege credentials strategies (AWS STS or GCP Workload Identity). This will require manual cherrypicks because the supported status of these features is not consistent across 4.7+

### CODE REVIEW

**Review by akhil-rane — COMMENTED** [review]
/lgtm

**Comment by jeana-redhat** [comment]
@jianping-shu ready for verification, PTAL

**Comment by jianping-shu** [comment]
Looks good to me, LGTM

File: authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@ toc::[]
 
 Manual mode with GCP Workload Identity is supported for Google Cloud Platform (GCP).
 
+[NOTE]
+====
+This credentials strategy is only supported for new {product-title} clusters and must be configured during installation. You cannot reconfigure an existing cluster that uses a different credentials strategy to use this feature.
```
Comment (kalexand-rh) [inline authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.adoc:13]:
```suggestion
This credentials strategy is supported for only new {product-title} clusters and must be configured during installation. You cannot reconfigure an existing cluster that uses a different credentials strategy to use this feature.
```

**Review by kalexand-rh — COMMENTED** [review]
I have a suggestion, then it LGTM!

**Pull request merged**